pkg/monitoring/rules/recordingrules/vmi.go

```diff
@@ -23,6 +23,14 @@ import (
 )
 
 var vmiRecordingRules = []operatorrules.RecordingRule{
+	{
+		MetricsOpts: operatormetrics.MetricOpts{
+			Name: "kubevirt_vmi_phase_count",
+			Help: "Sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`].",
+		},
+		MetricType: operatormetrics.GaugeType,
+		Expr:       intstr.FromString("sum by (node, phase, os, workload, flavor, instance_type, preference) (kubevirt_vmi_info)"),
```

Comment:
@machadovilaca I think you are missing here the new OS details that Assaf added.
Also for the VMI metric.

Reply:
why do we need the guest details in the aggregation metric?

Reply:
added